### ISSUE TITLE
Incompatible units: 'em' and ''.

### ISSUE THREAD

**Issue by ajorgensen** [thoughtbot/neat#432]
I am using bourbon 4.2.6 and neat 1.7.4. At the top of my applcation.scss file i have:

```
@import "bourbon/bourbon";
@import "neat/neat";
```

The stack trace is:
```
Extracted source (around line #37):
  @if $increment > 0 {
    @for $i from 1 through $increment {
      @if $double-stranded and ($v1 * $ratio) > $v2 {
        $value: $v2;
        $v2: ($v2 * $ratio);
```

![image](https://cloud.githubusercontent.com/assets/509779/13768187/64781ade-ea49-11e5-8355-ccb47e64fe2f.png)

I looked through the issues and saw this cropped up a few times but none of solutions in there worked for me.

**Comment by ajorgensen** [comment]
So the fix was to downgrade bourbon from 5.0.0.beta.4 by deleting the bourbon directory and running `bundle exec bourbon install` again.

**Comment by tysongach** [comment]
Sounds like this got resolved. Let us know if you have any other questions or issues.
